test/sanity/ignore.txt

```diff
@@ -248,6 +248,7 @@ lib/ansible/plugins/doc_fragments/validate.py metaclass-boilerplate
 lib/ansible/plugins/lookup/sequence.py pylint:blacklisted-name
 lib/ansible/plugins/strategy/__init__.py pylint:blacklisted-name
 lib/ansible/plugins/strategy/linear.py pylint:blacklisted-name
+lib/ansible/utils/collection_loader/_collection_finder.py pylint:undefined-variable  # we're falling back to the Python 2.x reload global function, which trips this error on 3.x
```

Comment:
I'd prefer an inline pylint ignore comment in the source instead of ignoring it here.

Reply:
I saw a bunch of the pycharm `#noinspection` ones in the same areas, but pylint doesn't seem to pay attention to those (?), so I did a couple of the pylint-specific ones too. Makes sense though, so they're attached to the problem line(s) rather than file-wide.